src/transformers/__tests__/workpresentation.auto.js

```diff
@@ -53,9 +56,9 @@ const expected = {
 const context = {
   services: {
     ddbcmsapi: 'https://cmscontent.dbc.dk/',
-    moreinfo: 'https://moreinfo.addi.dk/2.11/',
+    moreinfo: 'https://moreinfo.addi.dk/2.10/',
```

Comment:
den forstår jeg ikke - 2.10 findes såvidt jeg ved ikke - det gør 2.11

Reply:
Det må være min context som er lidt bagud, det betyder dog ikke noget for denne test, andet end det er lidt forvirrende.